plugins/fuse/server/server_fuse.py

```diff
@@ -121,7 +121,8 @@ def _getPath(self, path):
         except ValidationException:
             raise fuse.FuseOSError(errno.EROFS)
         except Exception:
-            self.log.exception('ServerFuse server internal error')
+            if self.log:
+                self.log.exception('ServerFuse server internal error')
```

Comment:
Out of scope of the PR, but would it make more sense to assume `log` is always a logger? Seems easier to enforce that and control logging through configuration rather than always having to use `if self.log`.

Reply:
Yes.  This change was for consistency with the rest of the logging -- a future PR could just remove these clauses and add a proviso that `self.log` must be a logger.